test/functional/cmdLineTests/criu/criuRandomScript.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+#
+# Copyright (c) 2022, 2022 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+#
+
+echo "start running script"
+if [ "$4" == "Checkpoint" ]
+then
+    # the expected arguments are:
+    # $1 is the TEST_ROOT
+    # $2 is the JAVA_COMMAND
+    # $3 is the JVM_OPTIONS
+    $2 $3 -XX:+EnableCRIUSupport -cp $1/criu.jar CRIURandomTest >>testOutput 2>&1
```

Comment:
Why does this append (`>>`) instead of just write (`>`) to `testOutput`?

Reply:
Your response (https://github.com/eclipse-openj9/openj9/pull/14824#discussion_r869571363) would have been better placed here.

With just `>` only the content from the *last* process would be in the file (not the first). Please add a comment explaining why append is appropriate here.

Reply:
Strange, my two most recent comments went to the wrong threads. It was meant for here.
Yes, I meant the last restore. Will add a comment.